backend/pool/pool.js

```diff
@@ -1,6 +1,6 @@
-const {sample, shuffle, random, range, times, constant, pull} = require("lodash");
+const { sample, shuffle, random, range, times, constant, pull } = require("lodash");
```

Comment:
this file rename is a little confusing to me.
what is pool/pool? should it be `pool/index.js`?

Reply:
I moved this into /pool because I am hoping to make /pool a component (module? package? not sure on the terminology here) that exposes PoolBuilder as an interface.

I'm happy to go with whatever the convention is for node.js (I don't really have any node.js experience).

Re: pool/pool, please see my notes about where I see this change going in the pull request description.

I'm hoping to see PoolBuilder being split into 3 different implementations of an abstract interface (cube, standard, chaos) and all the logic currently in pool/pool would be split up and included directly in those 3 interface implementations.

Reply:
So I did a bit more research here and I see that index.js serves as the entry point for passing a folder into require(). I don't think this file makes sense as an entry point to the 'pool' abstraction (indeed it is not included by anything external to this folder anymore).

I can't think of anything useful an index file would do for this folder, I think it would just make the import in game.js more confusing probably? Please let me know what you think :)